src/pybind/mgr/nfs/export.py

```diff
@@ -138,6 +157,22 @@ def check_user_config(self) -> bool:
         return False
 
 
+def nfs_rados_configs(rados: 'Rados', nfs_pool: str = POOL_NAME) -> Set[str]:
+    """Return a set of all the namespaces in the nfs_pool where nfs
+    configuration objects are found. The namespaces also correspond
+    to the cluster ids.
+    """
+    ns: Set[str] = set()
+    prefixes = (EXPORT_PREFIX, CONF_PREFIX, USER_CONF_PREFIX)
+    with rados.open_ioctx(nfs_pool) as ioctx:
+        ioctx.set_namespace(LIBRADOS_ALL_NSPACES)
+        for obj in ioctx.list_objects():
+            log.info("obj: %r", obj)
```

Comment:
What's the reason for printing every object in the list to info logs? Are we expecting cases where somebody runs this and then wants to see a list of all the objects we checked somewhere?

Reply:
gah. I swore I removed that.

Reply:
now its really removed